pumps/hybrid.go

```diff
@@ -70,7 +70,7 @@ type HybridPumpConf struct {
 	ConnectionString string `mapstructure:"connection_string"`
 	// Your organisation ID to connect to the MDCB installation.
```

Comment:
@dcs3spp should organisation be changed to organization?

Reply:
correct. We haven't done this word yet but we can add it @dcs3spp

Reply:
@letzya @sredxny I've updated organisation to organization